hooks/build

```diff
@@ -4,7 +4,8 @@
 # $IMAGE_NAME var is injected into the build so the tag is correct.
 
 docker build \
-		--build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
-		--build-arg VCS_REF=`git rev-parse --short HEAD` \
-		--build-arg VERSION=$(cat VERSION) \
-		-t $IMAGE_NAME .
+		--cache-from=${IMAGE_NAME} \
```

Comment:
I think there should be also a docker pull before the build so with this we can ensure that the cache exists

```
docker pull ${IMAGE_NAME}
```

Reply:
Added `--pull` to the build command